src/utils/system_utils/src/system_utils.c

```diff
@@ -19,6 +19,7 @@
 
 #include <aduc/string_c_utils.h>
 #include <azure_c_shared_utility/strings.h>
+#include <dirent.h>
```

Comment:
Likely can remove this line. I believe it's causing windows build pipeline to fail:

https://dev.azure.com/azure-device-update/adu-linux-client/_build/results?buildId=14245&view=logs&j=2dd7e357-93c9-5cd9-3155-fb06f1278828&t=185a2715-b6bc-50a7-3503-0bdaa3b94916&l=744

D:\a\_work\1\s\src\utils\system_utils\src\system_utils.c(22,10): fatal  error C1083: Cannot open include file: 'dirent.h': No such file or directory [D:\a\_work\1\s\out\Debug\src\utils\system_utils\system_utils.vcxproj]

ERROR: Build failed (Error 1)

Reply:
Other option is to add this to libaducpal to handle management.